app/packs/src/fetchers/SamplesFetcher.js

```diff
@@ -184,14 +184,21 @@ export default class SamplesFetcher {
     }).then((response) => {
       return response.json();
     }).then((json) => {
-      for (let i = 0; i < json.error_messages.length; i++) {
+      if (Array.isArray(json.error_messages)) {
+        for (let i = 0; i < json.error_messages.length; i++) {
```

Comment:
maybe with a lamba for better readability?
`
json.error_messages.forEach(
(errorText)=>  NotificationActions.add({message: errorText, level: 'error', autoDismiss: 10});)
`